docs/crio.conf.5.md

```diff
@@ -99,7 +99,7 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
   The _name_ of the OCI runtime to be used as the default.
 
 **default_ulimits**=[]
-  A list of ulimits to be set in containers by default, specified as "<ulimit name>=<soft limit>:<hard limit>", for example:"nofile=1024:2048". If nothing is set here, settings will be inherited from the CRI-O daemon.
+  A list of ulimits to be set in containers by default, specified as "<ulimit name>=<soft limit>:<hard limit>", for example:"nofile=1024:2048". If nothing is set here, limits that aren't explicitly set by runtime-tools will be inherited from the process launching the container.
```

Comment:
It's not wrong, but I've found it better to limit contractions, especially ones that are less frequently used, in man pages.  They are perfectly correct but less understandable for those who did not grow up speaking English.  My $.02.
```suggestion
  A list of ulimits to be set in containers by default, specified as "<ulimit name>=<soft limit>:<hard limit>", for example:"nofile=1024:2048". If nothing is set here, limits that are not explicitly set by runtime-tools will be inherited from the process launching the container.
```